src/argilla/client/feedback/training/schemas.py

```diff
@@ -154,49 +159,74 @@ def for_text_classification(
             >>> dataset = rg.FeedbackDataset.from_argilla(name="...")
             >>> task = TrainingTask.for_text_classification(
             ...     text=dataset.fields[0],
-            ...     label=dataset.questions[0],
+            ...     label=dataset.questions[0]
             ... )
             >>> dataset.prepare_for_training(framework="...", task=task)
 
+            >>> from argilla import LabelQuestion, TrainingTask
+            >>> from collections import Counter
+            >>> def formatting_func(sample: Dict[str, Any]) -> Union[Tuple[str, str], Tuple[str, List[str]]]:
+            ...     text = sample["text"]
+            ...     values = [annotation["value"] for annotation in sample["label"]]
+            ...     counter = Counter(values)
+            ...     if counter:
+            ...         most_common = counter.most_common()
+            ...         max_frequency = most_common[0][1]
+            ...         most_common_elements = [element for element, frequency in most_common if frequency == max_frequency]
+            ...         label = random.choice(most_common_elements)
+            ...         return (text, label)
+            ...     else:
+            ...         return None
```

Comment:
IMO we could showcase a simpler example here e.g.

```suggestion
            >>> def formatting_func(sample):
            ...     return f"Text: sample['text']"
```

Reply:
this is a simple most common counter. A simpler example would be just taking a random example?

Reply:
@tomaarsen what do you think about this?